main/test/flix/Test.Exp.Lambda.ExtMatch.flix

```diff
@@ -0,0 +1,45 @@
+mod Test.Exp.Lambda.ExtMatch {
+
+    @test
+    def testLambdaExtMatch01(): #| A(Int32) |# -> Int32 = ematch A(x) -> x
+
+    @test
+    def testLambdaExtMatch02(): #| A(Int32, Int32) |# -> Int32 = ematch A(x, y) -> ((x + y) : Int32)
+
+    @test
+    def testLambdaExtMatch03(): #| A(Int32, Int32, Int32) |# -> Int32 = ematch A(x, y, z) -> ((x + y + z) : Int32)
+
+    @test
+    def testLambdaExtMatch04(): #| A(Int32, Int32, Int32, Int32) |# -> Int32 = ematch A(x, y, z, _) -> ((x + y + z) : Int32)
+
+    @test
+    def testLambdaExtMatch05(): #| A(Int32) |# -> Int32 = ematch _ -> 42
+
+    @test
+    def testLambdaExtMatch06(): #| A(Int32), B(Bool) |# -> Int32 = ematch _ -> 42
+
+    @test
+    def testLambdaExtMatch07(): Int32 = (ematch A(x) -> x)(xvar A(1))
```

Comment:
I would be better if these checked the result.

Reply:
E.g. returned a Boolean via comparison to the expected result ==

Reply:
Roger